src/graphics/texture.js

```diff
@@ -545,6 +554,33 @@ Object.assign(pc, function () {
         }
     });
 
+    Object.defineProperty(Texture.prototype, 'rgbm', {
```

Comment:
you could use Object.defineProperties (see mesh.js) to group them together.

Reply:
I was going to submit a separate PR changing all texture properties, but I've just done it here instead.